.drone.star

```diff
@@ -2053,6 +2054,32 @@ def listRemoteCache():
         ],
     }]
 
+def stopBuild():
+    return [{
+        "name": "stop-build",
+        "image": "drone/cli:alpine",
+        "pull": "always",
+        "settings": {
+            "server": "https://drone.owncloud.com/",
+            "token": "1efGWn17Riebj5qa4UW4oNaQijtAnONQ",
```

Comment:
Please do not expose these tokens in code, you can rather use drone secrets for this.

Reply:
Okay. I actually wanted to see if that works. I will make the changes